app/models/emis_redis/veteran_status.rb

```diff
@@ -3,6 +3,17 @@
 require 'emis/veteran_status_service'
 
 module EMISRedis
+  # Much of this class depends on the Title 38 Status codes, which are:
```

Comment:
@patrickvinograd @lihanli - I added you two as reviewers because you were the two devs that have worked with this `app/models/emis_redis/veteran_status.rb` file.

As you'll see below, all I have done is:

- refactored some of the `title38_status` into a private method for reusability 
- added three new helper methods to access veteran status related data from Redis

You'll note that everything is still leveraging the `emis_response` method, thereby using Redis first, before making external service calls.